wcd2reshare.py

```diff
@@ -60,20 +73,9 @@ def lambda_handler(event: dict, context: dict) -> dict:
     Reshare / Vufind. Return an object representing an HTTP redirect response to
     ReShare / Vufind, using the query string if one was built.
     """
-    logger = logging.getLogger(__name__)
-    logging.basicConfig(level=logging.INFO)
-    env = os.environ["WORKSPACE"]
-    if sentry_dsn := os.getenv("SENTRY_DSN"):
-        sentry_sdk.init(
-            dsn=sentry_dsn,
-            environment=env,
-            integrations=[
-                AwsLambdaIntegration(),
-            ],
-        )
-        logger.info(
-            "Sentry DSN found, exceptions will be sent to Sentry with env=%s", env
-        )
+
+    if env is None:
```

Comment:
I think we should raise an exception here instead of just logging a warning. It will be much easier to spot that something is wrong if the function totally fails to run, plus if WORKSPACE is actually a required env variable then we should enforce that requirement.

Reply:
`TypeError` or `ValueError` or something else?

Reply:
Also, the sentry init has a param for the env, but I believe that's optional. I don't use the WORKSPACE env anywhere else, so I'm not sure if it is really needed. I was just copying from what Eric pointed me to.

Reply:
Right. The reason we supply the env to sentry init is that we handle sentry alerts differently for different environments. We can see Sentry exceptions from dev and stage in the Sentry UI, but we don't want to get emailed alerts for them. By sending the env, we can tell Sentry to only send notifications for exceptions raised in the "prod" environment. That way we don't get a bunch of spam when we're trying out new code in other environments. So, although it's not a required param from the sentry side, it is required by us for our apps. And I would make it a `ValueError`.